src/Api/Management/Entity/Developer.php

```diff
@@ -44,6 +44,13 @@ class Developer extends AppOwner implements DeveloperInterface
     /** @var string[] */
     protected $companies = [];
 
+    /**
+    * The original email address of the developer.
+    *
+    * @var string|null
+    */
+    protected $originalEmail;
```

Comment:
Since here a public getter for this property, this property should be `private`, because no subclass should modify this directly. The related business logic is managed by the `setEmail()` class and both child classes and external caller should only use that public interface.

Reply:
@mxr576 Fixed,
You are right, Thanks!